main.rb

```diff
@@ -0,0 +1,3 @@
+require_relative 'app'
+app = App.new
+app.run
```

Comment:
- [ ] As mentioned by the previous reviewer and for the project requirements, kindly ensure that your app is not crashing while testing it. I recommend you fix all errors as now your app is still showing the below error while trying to test it. Nothing is running
![image](https://github.com/hafiz1379/Catalog-of-my-things/assets/83042742/c4ee9b12-47bd-481e-a297-7320d3b4c774)